### PR TITLE
styled footer, nav + search on display

### PR DESCRIPTION
**Pull request by tsvetelinak0**
### What change does this PR introduce?

Added new styles to navbar, footer and a little bit of tidying the code

### Notes

Search container on display but not styled

### Ticket

- [Trello Ticket](https://trello.com/c/UkBdwBxp/65-create-simple-footer-html)

### Screenshots

Screenshot 2020-04-20 at 17 04 35

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[[source]]
```
Comment (lili2311) [inline Pipfile:1]: why did this sneak in? We have a requirements.txt file for Python dependencies :)

**Review by lili2311 — CHANGES_REQUESTED** [review]

File: frontend/src/styles/base/_variables.scss [hunk 2]
```diff
@@ -79,12 +82,13 @@ $h1-margin-top: (
...
     lg: 40px,
 ) !default;
-$h2-font-size: (
-    base: 26,
-    md: 32,
-    lg: 36,
-    xl: 36,
-) !default;
+$h2-font-size: 36px;
+// $h2-font-size: (
```
Comment (lili2311) [inline frontend/src/styles/base/_variables.scss:86]: please delete un-commented out code if unused, this looks like it was meant to set size per breakpoint is there something doing this?

File: templates/layout.html [hunk 3]
```diff
@@ -1,32 +1,27 @@
...
-        <div class=footer-nav>
-            <a href="{% url 'homepage' %}" class='navbar_header--item'>Home</a>
-			<a href="{% url 'about' %}" class='navbar_header--item'>About</a>
-            <a href="{% url 'contact' %}" class='navbar_header--item'>Contact</a>
-        </div>
-        <p class=created>Created by Polyglot Devs London</p>
-        <a href="https://twitter.com/devs_london">@devs_london</a> 
+    </header>
 
+    {% block search %}
```
Comment (lili2311) [inline templates/layout.html:13]: we do not use the blocks, we only have `main` please keep all changes inside main. You will need to re-base this branch to make sure you have the latest develop changes.

**Comment by lili2311** [comment]
👋 I enabled heroku review apps and it looks like this PR breaks the front end styles we had for the search? https://sharespots-feat-style-f-yrqepy.herokuapp.com/

File: Pipfile (hunk 1 above)
Comment (tsvetelinak0) [reply Pipfile:1]: Pipfile was removed in next PR. sneaked by mistake as create venv with pyenv

Comment (lili2311) [reply Pipfile:1]: hehe, please delete here too as this may get merged first. Don't want to confuse Heroku when we deploy

**Pull request closed without merging**